js/RateCurve.js

```diff
@@ -75,8 +79,7 @@ RateCurve.prototype.rcCommandRawToDegreesPerSecond = function (rcData, rate, rcR
             angleRate = (rate + 27) * inputValue / 16;
```

Comment:
Is this line only for the old situation?

I was trying to find a new anglerate calculation but couldnt.

This line reflects 205deg/sec as it was before (integer legacy stuff). I changed this to be 200deg/sec to make the math a bit easier.